docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to use `earthdata-varinfo` to publish UMM-Var entries to CMR"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Version 2.0.0"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview:"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This version of `earthdata-varinfo` adds functionality to publish records to CMR, along with a single overarching function that wraps the search, download and publication functionality into a single function for the convenience of the end-user. Additionally, the function signatures for `cmr_search.get_granules` and `cmr_search.download_granule` have been updated to accept the full Authorization header, instead of a bearer token, so that they are also compatible with LaunchPad tokens."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up your environment to run this notebook"
```

Comment:
It's probably worth a quick sentence to mention that a user should initially set up and activate a Python environment with either `pyenv` or conda before running any `pip` commands.

Reply:
* [ ] Add comments about activating python environment with pyenv or conda before running any pip commands